edu-2523-update-marketplace-content (#327)
aziontech/docs: src/content/docs/en/pages/marketplace/permissions-marketplace.mdx

---
title: Azion Marketplace's permissions
description: Permissions needed to install Azion Marketplace solution.
meta_tags: permissions, help, edge computing
namespace: documentation_permissions_mktp

> permalink: /documentation/products/guides/permissions-mktp/

Author: ```suggestion
permalink: /documentation/products/guides/permissions-marketplace/
```

---

import Button from '~/components/Button.astro';

To install and deploy solutions from **Azion Marketplace**, you could have to grant privileges for the **Marketplace** to read and write the configurations of your edge applications.

---

## Common permissions for integrations

Every integration has its own needs for privileges. For example, to read and write the **Rules Engine** of your edge application in order to create a rule to trigger the execution of the integration.

### Main Settings

The **Main Settings** tab on your edge application includes the main configurations needed to execute your integrations. When a integration asks for privileges to write and read the **Main Settings**, this means this integration needs to read and/or write one of the following configurations on your edge application:

- **Functions**

>     - Must be enabled in order to execute any integration from **Marketplace**. To do so, it's mandatory to allow access if you want to run any integration installed via **Marketplace**. It's used to change the viewer request/response, customize arguments (**Args**) for your functions, and choose *criteria* to execute in the **Rules Engine**.

Reviewer: ```suggestion
    - Must be enabled in order to execute any integration from **Marketplace**. To do so, it's mandatory to allow access if you want to run any integration installed via **Marketplace**. It's used to change the viewer request/response, customize arguments (**Args**) for your functions, and choose *criteria* to execute in **Rules Engine**.
```

- **Application Acceleration**

>     - This option will be mandatory if your integration needs to use advanced rules on **Rules Engines**, such as `Cache Key`, `Bypass Cache`, `Forward Cookies`, or `Proxy methods`.

Author: ```suggestion
    - This option will be mandatory if your integration needs to use advanced rules on **Rules Engines**, such as `Bypass Cache` or `Forward Cookies`.
```

- **Image Processor**
    - It helps to optimize the experience with images on your edge application.
- **Load Balancer**
    - A traffic balancer to assure you the best-on-class reliability and network congestion control.

:::note
Some integrations on Azion Marketplace may require the L2 Caching module. Contact the [Azion Sales team](https://www.azion.com/en/contact-sales/) to activate this module for your account.
:::

### Rules Engine

> Every edge application relies on the **Rules Engine** in order to execute the *behavior* and *criteria* you need to run the application on the edge. For this, Azion **Marketplace** needs the privileges to read and write your rules on Rules Engine.

Reviewer: ```suggestion
Every edge application relies on **Rules Engine** in order to execute the *behavior* and *criteria* you need to run the application on the edge. For this, Azion **Marketplace** needs the privileges to read and write your rules on Rules Engine.
```

### Edge Firewall

> Some integrations will run on the **Edge Firewall** instance. These integrations will need more privileges than granted to an edge application. To know the **Edge Firewall** access privileges, visit the guide [Access Permissions for Edge Firewall](/en/documentation/products/edge-firewall/access-permissions/).

Author: ```suggestion
Some integrations will run on the **Edge Firewall** instance. These integrations will need more privileges than granted to an edge application. To know the **Edge Firewall** access privileges,  visit the documentation for [Teams Permissions](/en/documentation/products/accounts/teams-permissions/).
```
